src/crt.h

```diff
@@ -28,6 +28,7 @@ typedef struct _aws_crt_resource {
  * NOTE: Whatever structure is passed in here MUST have an aws_crt_resource as its first member
  */
 void *aws_crt_resource_new(void *object);
+void aws_crt_resource_init(aws_crt_resource *resource);
```

Comment:
These are "private helper" functions for use within the .c files of aws-crt-ffi. I doubt they will be the last. I recommend moving them to a private header

Reply:
This IS a private header :)